requirements.txt

```diff
@@ -17,6 +17,7 @@ pymacaroons==0.9.2; sys_platform != 'win32'
 pymacaroons==0.10.0; sys_platform == 'win32'
 pymacaroons-pynacl==0.9.3
 pysha3==1.0b1; python_version < '3.6'
+raven==6.5.0
```

Comment:
I think it would be good to explain somewhere, maybe in the description of this pr, why raven is not a requirement for the deb.

Reply:
In the PR now